### DISCUSSION TITLE
How to export pretrained model to .pb?

### DISCUSSION THREAD

**Discussion by SSsriranga** [talmolab/sleap#1309] [Help!] [answered]
I have a  pretrained (UNet) Single-Instance model and have been setting up the Bonsai interface for SLEAP for real-time tracking; All the necessary packages are installed as per the Bonsai.SLEAP git repo, but I'm not sure how to export the pretrained SLEAP model to .pb format. I have tried converting the .h5 (model) file using some general utilities functions from tensorflow/keras but none have worked, and I don't see any .pb conversion utility functions in the SLEAP repo either -- do you have any suggestions for how this can be done? 

Thanks for your help!

**Comment by roomrys** [comment, marked as answer]
Hi @SSsriranga,

You can export your SLEAP model to a .pb through the [sleap-export](https://sleap.ai/guides/cli.html#sleap-export) CLI. We will be updating our documentation on this to be more explicit.

Thanks,
Liezl

Related: #1074
